gui/backend/send_metric_event.go

```diff
@@ -0,0 +1,42 @@
+package backend
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type sendMetricEventRequest struct {
+	EventType  string      `json:"type"`
+	EventProps interface{} `json:"props"`
```

Comment:
and `event_props` -- helps to distinguish between `user_props`, since this will not be merged with user props right now